src/main/java/uk/gov/hmcts/cmc/claimstore/config/properties/pdf/DocumentTemplates.java

```diff
@@ -15,6 +15,10 @@
         return readBytes("/citizen/templates/document/claimIssueReceipt.html");
     }
 
+    public byte[] getClaimantResponseCountyCourtJudgementDetails() {
```

Comment:
That name is misleading. We should have something like `getCountyCourtJudgmentByAdmission`